modules/virt-querying-metrics.adoc

```diff
@@ -5,105 +5,114 @@
 [id="virt-querying-metrics_{context}"]
 = Virtualization metrics
 
-The following metric descriptions include example Prometheus Query Language (PromQL) queries.
+The following metric descriptions include example Prometheus Query Language (PromQL) queries. These metrics are not an API and might change between versions.
 
 [NOTE]
 ====
-These metrics are not an API and might change between versions.
+The following examples use `topk` queries that specify a time period. If virtual machines are deleted during that time period, they can still appear in the query output.
 ====
 
 [id="virt-promql-vcpu-metrics_{context}"]
 == vCPU metrics
 
-[NOTE]
-====
-The vCPU metric requires the `schedstats=enable` kernel argument applied to the `MachineConfig` object before it can be used. This kernel argument enables scheduler statistics used for debugging and performance tuning and adds a minor additional load to the scheduler.
-====
+The following query can identify virtual machines that are waiting for Input/Output (I/O):
 
 `kubevirt_vmi_vcpu_wait_seconds`::
 Returns the wait time (in seconds) for a virtual machine's vCPU.
 
-A value above '0' means that the vCPU wants to run, but the host scheduler cannot run it yet. This inability to run indicates that there is an issue with Input/Output.
+A value above '0' means that the vCPU wants to run, but the host scheduler cannot run it yet. This inability to run indicates that there is an issue with I/O.
 
-.Example query
+.Example vCPU wait time query
 [source,promql]
 ----
-topk(3, sum by (name, namespace) (round(irate(kubevirt_vmi_vcpu_wait_seconds[6m]), 0.1))) > 0
+topk(3, sum by (name, namespace) (round(irate(kubevirt_vmi_vcpu_wait_seconds[6m]), 0.1))) > 0 <1>
 ----
-The above query returns the top 3 VMs waiting for I/O at every given moment in time over the time period.
+<1> This query returns the top 3 VMs waiting for I/O at every given moment over a six-minute time period.
+
+[NOTE]
```

Comment:
Because this note is kind of a prerequisite, would it be more useful to place it above the example query?